.github/workflows/cd.yml

```diff
@@ -98,7 +98,7 @@ jobs:
         id: version-change
         run: |
           echo "::set-output name=CHANGED::$(git tag --points-at HEAD | xargs)"
-          echo "::set-output name=TEMPLATE_VERSION::$(git tag --points-at HEAD | grep templates)"
+          echo "template_ver=$(git tag --points-at HEAD | grep templates@)" >> $GITHUB_ENV
```

Comment:
use template_ver as an env variable, this variable may be empty in this step by conventional-commits if templates has not any  code change